### PR TITLE
Covid-19 services post migration update

### PR DESCRIPTION
**Pull request by gclssvglx**
## What

Update covid-19-services documentation to reflect the changes made during the migration process of the find support service to smart answers.

[Trello](https://trello.com/c/7yaKhyf0/544-update-the-developer-docs)

### CODE REVIEW

**Review by theseanything — DISMISSED** [review]

File: source/manual/covid-19-services.html.md [hunk 1]
```diff
@@ -4,416 +4,115 @@ title: COVID-19 Services
...
 tell us if they need help accessing essential supplies and support.
 Users will have received a link to the service in a letter or a text
 message from the NHS, or been advised by their GP to fill in the form.
-They can fill the form in themselves, or someone else can fill it in
-for them.
 
-There is also an interactive voice response (IVR) automated phone
-service provided by AWS. This is for users who do not have access to
-the internet.
+**The service has been rebuilt by #vulnerable-people-services team.**
```
Comment (reggieb) [inline source/manual/covid-19-services.html.md:55]: Very minor (addition of "the"):
```
**The service has been rebuilt by the #vulnerable-people-services team.**
```

File: source/manual/covid-19-services.html.md [hunk 2]
```diff
@@ -4,416 +4,115 @@ title: COVID-19 Services
...
-- [Start page](https://www.gov.uk/coronavirus-support-from-business)
-- [Service](https://coronavirus-business-volunteers.service.gov.uk/medical-equipment)
-- [GitHub](https://github.com/alphagov/govuk-coronavirus-business-volunteer-form)
+**The service has been replaced with a guidance page.**
+
+A backup of the production database was taken prior to the service
+being removed. It is available in the `govuk-production-database-backups`
+AWS S3 bucket named `coronavirus-business-volunteer-form/production.sql.gzip`
+with 365 day retention period (expiry date: Wed, 13 Oct 2021 00:00:00
+GMT) as per our privacy policy.
```
Comment (reggieb) [inline source/manual/covid-19-services.html.md:32]: This paragraph isn't very easy to read on the page as it contains long names and a date that cross line breaks. I'd suggest breaking it down into bullets. Something like:
```
A backup of the production database was taken prior to the service
being removed. It is available at:

- AWS S3 bucket - `govuk-production-database-backups`
- Key - `coronavirus-business-volunteer-form/production.sql.gzip`
- Retention period - 365 days (as per our privacy policy)
- expiry date - Wed, 13 Oct 2021 00:00:00 GMT
```

File: source/manual/covid-19-services.html.md [hunk 3]
```diff
@@ -4,416 +4,115 @@ title: COVID-19 Services
...
+
+For details...
```
Comment (reggieb) [inline source/manual/covid-19-services.html.md:34]: Would something a little more descriptive be helpful here. Something like:

```
The file can be retrieved with the following command
```
(I'm not sure if that command retrieves the file or just information about it)

File: source/manual/covid-19-services.html.md [hunk 4]
```diff
@@ -4,416 +4,115 @@ title: COVID-19 Services
...
-
-## Hosting
+**The service has now been migrated and is now a smart answer.**
 
-### PaaS
+A backup of the production database was taken prior to the service
+being removed. It is available in the `govuk-production-database-backups`
+AWS S3 bucket under `coronavirus-find-support/production.sql.gzip`
+with 365 day retention period (expiry date: Fri, 08 Oct 2021 00:00:00 GMT)
+as per our privacy policy.
```
Comment (reggieb) [inline source/manual/covid-19-services.html.md:74]: Again, I think this information would be more readable if it was presented in a bullet list:

```
A backup of the production database was taken prior to the service
being removed. It is available at:

- AWS S3 bucket - `govuk-production-database-backups`
- Key - `coronavirus-find-support/production.sql.gzip`
- 365 day retention period (as per our privacy policy)
- expiry date - Fri, 08 Oct 2021 00:00:00 GMT
```

File: source/manual/covid-19-services.html.md [hunk 5]
```diff
@@ -4,416 +4,115 @@ title: COVID-19 Services
...
 
-All applications are hosted on GOV.UK Platform as a Service (PaaS) in
-the Ireland region.  PaaS provides a scalable hosting platform based
-on CloudFoundry managed by GDS.  You can read more in the [PaaS
-technical documentation][paas-docs].
+For details...
```
Comment (reggieb) [inline source/manual/covid-19-services.html.md:76]: Again, I think a description of what the command does would be preferable to "For details.."

**Review by reggieb — COMMENTED** [review]
Looking good. A couple of small changes suggested.

File: source/manual/covid-19-services.html.md (hunk 1 above)
Comment (gclssvglx) [reply source/manual/covid-19-services.html.md]: Thanks @reggieb - fixed 👍

File: source/manual/covid-19-services.html.md (hunk 2 above)
Comment (gclssvglx) [reply source/manual/covid-19-services.html.md]: Thanks @reggieb - fixed 👍

File: source/manual/covid-19-services.html.md (hunk 3 above)
Comment (gclssvglx) [reply source/manual/covid-19-services.html.md]: Thanks @reggieb - fixed 👍

File: source/manual/covid-19-services.html.md (hunk 4 above)
Comment (gclssvglx) [reply source/manual/covid-19-services.html.md]: Thanks @reggieb - fixed 👍

File: source/manual/covid-19-services.html.md (hunk 5 above)
Comment (gclssvglx) [reply source/manual/covid-19-services.html.md]: Thanks @reggieb - fixed 👍

**Review by reggieb — APPROVED** [review]
Good stuff!

**Pull request merged**